collection/rancher/v2.x/windows-log-collector/README.md

```diff
@@ -0,0 +1,60 @@
+# rancher-windows-log-collector
+Windows log collector for Rancher Windows Worker Nodes
+
+
+## How To Run the Script
```

Comment:
For people who have little/no windows experience, might be worth mentioned that they need to open a PS or CMD or whatever type of window to run this in.

Reply:
I updated the instructions, mind giving them a +1? @dnoland1